src/js/WebParts/ResourceAllocation/index.tsx

```diff
@@ -290,7 +294,9 @@ export default class ResourceAllocation extends BaseWebPart<IResourceAllocationP
      * Fetches availability items from list on root
      */
     private async fetchAvailabilityItems(): Promise<Array<any>> {
-        const itemsAvailability = await sp.web.lists.getByTitle(__.getResource("Lists_ResourceAllocation_Title"))
+        const web = new Web(_spPageContextInfo.siteAbsoluteUrl);
```

Comment:
Hva gjør endringen her?

Reply:
Bruker `siteAbsoluteUrl` fremfor sp.web for å alltid hente listen fra rot